tests/org.pitest.pitclipse.ui.tests/src/org/pitest/pitclipse/ui/behaviours/pageobjects/PitSummaryView.java

```diff
@@ -46,4 +61,67 @@ public double getOverallCoverage() {
     public double getMutationCoverage() {
         return lastResults.getMutationCoverage();
     }
+
+    public void showViewIfNotOpen() {
+        if (summaryView == null) {
+            summaryView = bot.viewByTitle(SUMMARY_VIEW_TITLE);
+        }
+        summaryView.show();
+    }
+
+    public void clickButtonWithText(String text) {
+        showViewIfNotOpen();
+        toolbarButton(text).click();
+    }
+
+    /**
+     * Helper method, because other method works with tooltip. Gets the toolbar
+     * button matching the given toolbar button.
```

Comment:
Maybe this means that we should add tooltips on those buttons? Would that be useful?